specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/keys.json

```diff
@@ -1484,16 +1584,13 @@
       },
       "description": "The attributes of a key managed by the key vault service."
     },
-    "KeyBundle": {
+    "BaseKeyBundle": {
+      "type": "object",
       "properties": {
         "key": {
           "$ref": "#/definitions/JsonWebKey",
           "description": "The Json web key."
         },
-        "attributes": {
```

Comment:
This may lead to a breaking SDK change (not REST API-breaking). We'll have to account for that @vcolin7, @chlowell. We can wait and see. Different emitters deal with inheritance differently. It would break .NET if we actually generated code from swagger, which .NET does not (legacy code base before the track 2 phase 1 emitters were ready).